error: a function definition without a prototype is deprecated in all versions of C and is not supported in C2x [-Werror,-Wdeprecated-non-prototype]

Merged into master, but not in maint:
https://github.com/Gnucash/gnucash/pull/1513#issuecomment-1404466067

Index: gnucash/import-export/log-replay/gnc-log-replay.c
--- gnucash/import-export/log-replay/gnc-log-replay.c.orig
+++ gnucash/import-export/log-replay/gnc-log-replay.c
@@ -109,9 +109,7 @@ static char *olds;
    returning a 0 length valid string between two consecutive ocurence of delim.
    It will also return a 0 length string instead of NULL when it reaches the end of s
 */
-static char * my_strtok (s, delim)
-char *s;
-const char *delim;
+static char * my_strtok (char *s, const char *delim)
 {
     char *token;
     /*DEBUG("strtok(): Start...");*/
